Makefile

```diff
@@ -34,7 +34,7 @@ export DOCKER_FILE        = $(BUILD_DIR)/Dockerfile
 export DOCKER_IMAGE      ?= $(COMPONENT_NAME)
 export DOCKER_BUILDER    ?= docker
 
-BEFORE_SCRIPT := $(shell build/before-make.sh)
+KUBECONFIG?=./.kubeconfig
```

Comment:
Move kubeconfig under the project root instead of the _output/ because we want to run:

```
go test -c ./test/e2e -o _output/e2e.test
_output/e2e.test -test.v -ginkgo.v
```

At the same layer.